ui/js/actions/file_info.js

```diff
@@ -102,20 +102,32 @@ export function doDeleteFile(outpoint, deleteFromComputer, abandonClaim) {
       const fileInfo = byOutpoint[outpoint];
```

Comment:
Can all or most of the code inside of this if statement simply be a call to `doAbandonClaim`? Seems like some redundant logic.

Reply:
Done.